BLUEPRINT/reactor.py

```diff
@@ -702,7 +715,8 @@ def build_TF_coils(
                 f"|   subject to: {self.params.TF_ripple_limit} % ripple"
             )
 
-            self.TF.optimise()
+            self.TF.build(self.callbacks.get("TF"))
+        # CTM: Should we just be relying on this mode instead?
```

Comment:
```suggestion
        # NOTE: Should we just be relying on this mode instead?
```

Reply:
I think it's actually better to delete this entirely. It was a note to myself while working which won't be useful to anyone else.